addons/l10n_in/tests/test_hsn_summary.py

```diff
@@ -578,3 +579,68 @@ def test_l10n_in_hsn_summary_4(self):
             },
         ))
         self._assert_tests(tests)
+
+    def test_l10n_in_hsn_summary_5(self):
+        """ Test with discount on lines. """
+        move = self.env['account.move'].create({
```

Comment:
The whole test is designed to test both python and javascript side so you must absolutely not create an invoice like this.

Reply:
I agree that this test is designed to test both the Python and JavaScript sides. However, the problem is that the value going into the method `_l10n_in_get_hsn_summary_table` is wrong, which is why I created the invoice, so the method `_l10n_in_get_hsn_summary_table` gets called.

Please let me know if there is another way I can check without creating an invoice.

Reply:
@niai-odoo Add the discount to the lines provided to the method then :)